core/client/src/main/java/alluxio/client/netty/NettyClient.java

```diff
@@ -106,8 +105,8 @@ public void initChannel(SocketChannel ch) throws Exception {
   private static ChannelType getChannelType() {
     if (PACKET_STREAMING_ENABLED) {
       try {
-        EpollChannelConfig.class.getField("EPOLL_MODE");
-      } catch (NoSuchFieldException e) {
+        EpollChannelOption.class.getField("EPOLL_MODE");
+      } catch (Throwable e) {
         LOG.warn("EPOLL_MODE is not supported in netty with version < 4.0.26.Final.");
```

Comment:
I would probably simply say "Could not get field EPOLL_MODE.", but on a second thought, what you have is also okay.